test/Github/Tests/Integration/TestCase.php

```diff
@@ -36,25 +36,21 @@ public function setUp()
         $this->client = $client;
     }
 
-    protected function auth(Client &$client, $accountNumber = 1)
+    protected function auth(Client $client, $accountNumber = 1)
     {
-        try {
-            (new Dotenv(__DIR__.'/../../../../'))->load();
-            $method = getenv('GITHUB_AUTH_METHOD');
-            if (!getenv('GITHUB_AUTH_METHOD')) {
-                return;
-            }
-            switch ($method) {
-                case 'token':
-                    $client->authenticate($method, getenv('GITHUB_TOKEN_'.$accountNumber));
-                    break;
-                case 'login':
-                    $client->authenticate($method, getenv('GITHUB_USERNAME_'.$accountNumber),
-                        getenv('GITHUB_PASSWORD_{$accountNumber}'));
-                    break;
-            }
-        } catch (\Exception $e) {
-            error_log('Unable to authenticated', 0);
+        (new Dotenv(__DIR__.'/../../../../'))->load();
+        $method = getenv('GITHUB_AUTH_METHOD');
+        if (!getenv('GITHUB_AUTH_METHOD')) {
+            return;
```

Comment:
Maybe `$this->markSkipped()` here?

Reply:
Since the use of env variables only occurs in InvitationsTest.php, I believe it would make sense to move the entire auth logic to it's own setup method, including the markSkipped idea.